Core.ApplicationServices/Organizations/IOrganizationService.cs

```diff
@@ -21,7 +22,7 @@ public interface IOrganizationService
 
         Result<Organization, OperationFailure> CreateNewOrganization(Organization newOrg);
 
-        public Result<Organization, OperationError> GetOrganization(Guid organizationUuid);
+        public Result<Organization, OperationError> GetOrganization(Guid organizationUuid, OrganizationDataReadAccessLevel accessLevel = OrganizationDataReadAccessLevel.Public);
```

Comment:
gør den nullable i stedet for så man skal forholde sig til den. Default vil da være AllowReadAccess :-)

Reply:
![image](https://user-images.githubusercontent.com/39791946/124461292-b95f6d00-dd90-11eb-82df-eaf7ecbaeabf.png)
Er det sådan du mener?